netimpair.py

```diff
@@ -182,219 +166,264 @@ def netem(
             delay_jitter_corr=0,
             reorder_ratio=0,
             reorder_corr=0,
-            toggle=[1000000]):
-        assert subprocess.call(
-            shlex.split(
-                "tc qdisc add dev {} parent 1:3 handle 30: netem".format(
-                    self.nic))) == 0
-        while len(toggle) != 0:
-            impair_cmd = "tc qdisc change dev {} parent 1:3 handle 30: netem rate 1000mbit loss {}% {}% duplicate {}% delay {}ms {}ms {}% reorder {}% {}%"\
-                .format(self.nic, loss_ratio, loss_corr, dup_ratio, delay, jitter, delay_jitter_corr, reorder_ratio, reorder_corr)
-            print("Setting network impairment:")
+            toggle=None):
+        '''Enable packet loss.'''
+        if toggle is None:
+            toggle = [1000000]
+        self._check_call(
+            'tc qdisc add dev {0} parent 1:3 handle 30: netem'.format(
+                self.nic))
+        while toggle:
+            impair_cmd = 'tc qdisc change dev {0} parent 1:3 handle 30: ' \
+                'netem loss {1}% {2}% duplicate {3}% delay {4}ms {5}ms {6}% ' \
```

Comment:
We need to specify rate because on some older systems the command will not work without rate. Specifically CentOS